apps/finance/app/src/lib/date-utils.js

```diff
@@ -3,6 +3,6 @@ import { format } from 'date-fns'
 export const MMDDYY_FUNC_FORMAT = 'MM/dd/yy'
```

Comment:
Wondering if we can remove this now, and avoid exporting the other format?

Reply:
Yep, we can actually do this after standardizing to `yyyy-MM-dd`.